### PR TITLE
merge questions 2.1 and 2.8

### PR DESCRIPTION
**Pull request by pes10k**
fixes issue #97

Since 2.8 and 2.1 cover a lot of the same subject matter, this PR moves the implied-enumeration from 2.8 into 2.1

***
<a href="https://pr-preview.s3.amazonaws.com/w3ctag/security-questionnaire/pull/100.html" title="Last updated on Dec 8, 2021, 9:50 PM UTC (2e3e988)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3ctag/security-questionnaire/100/7df4dce...2e3e988.html" title="Last updated on Dec 8, 2021, 9:50 PM UTC (2e3e988)">Diff</a>

### CODE REVIEW

**Comment by samuelweiler** [comment]
You are intending to drop the BEACON example, right?

In any case: mark the sub-questions as questions using the familiar "?".

**Comment by pes10k** [comment]
re https://github.com/w3ctag/security-questionnaire/issues/97#issuecomment-698624428, i will rework this PR to see if we need either section

File: index.bs [hunk 1]
```diff
@@ -101,6 +101,28 @@ See also
...
+about the browser user's environment (e.g., operating system configuration,
+browser configuration, hardware capabilities), and the user's prior activities
+and interests (e.g., browsing history, purchasing preferences, personal
+characteristics).
+
+1.  What information does your spec expose to the **first party** that
+    the **first party** cannot currently easily determine.
+2.  What information does your spec expose to **third parties** that
+    the **third parties** cannot currently easily determine.
+3.  What *potentially sensitive information* does your spec expose to the
```
Comment (plinss) [inline index.bs:120]: Should this be `*potentially identifying information*`?

Comment (pes10k) [reply index.bs]: Good catch, fixed!

**Comment by pes10k** [comment]
Okie dokie, how do things look now.  Cleaned up an additional typo too. Good to merge?

**Review by hober — APPROVED** [review]
Looks good to me!

**Pull request merged**